Mathlib/Analysis/Complex/UpperHalfPlane/Basic.lean

```diff
@@ -203,5 +203,16 @@ theorem vadd_im : (x +ᵥ z).im = z.im :=
   zero_add _
 
 end RealAddAction
+section complexUpperHalfPlane
+
+/-- The UpperHalfPlane as a subset of `ℂ`. This is convinient for takind derivatives of functions
+on the upper half plane. -/
+abbrev complexUpperHalfPlane := {z : ℂ | 0 < z.im}
```

Comment:
I don't like the name, it would be better to make clear what's different between this and `ℍ`. Perhaps `upperHalfPlaneSet`?